eth/vm/base.py

```diff
@@ -455,11 +469,13 @@ def execute_bytecode(self,
         )
 
         # Execute it in the VM
-        return self.state.get_computation(message, transaction_context).apply_computation(
-            self.state,
-            message,
-            transaction_context,
-        )
+        with self.state.trace(tracer):
```

Comment:
I think this call to `trace` is unnecessary. Neither `get_computation_class` nor `apply_computation` use `state`'s tracer attribute.